### PR TITLE
Create, read, update services for teachers

### PR DESCRIPTION
**Pull request by aytiqaqash**
Description:
No validations for this task

### CODE REVIEW

**Review by ttayping — APPROVED** [review]
why teacher model hasn't name and contact field?

File: src/main/java/com/webperside/courseerpbackend/repository/TeacherRepository.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import java.util.List;
+import java.util.Optional;
+
+@Mapper
+public interface TeacherRepository {
+    void insert(Teacher teacher);
+
+    void update(Teacher teacher);
+
+    Optional<Teacher> findById(Long id);
```
Comment (gadirovakhadija) [inline src/main/java/com/webperside/courseerpbackend/repository/TeacherRepository.java:15]: Inside the findById to be good to use @Param("id") annotation;

Comment (hamidsultanzadeh) [reply src/main/java/com/webperside/courseerpbackend/repository/TeacherRepository.java:15]: actually it is not required to use @Param. my batis will use field name by default, if you don't specify a name.

Comment (hamidsultanzadeh) [reply src/main/java/com/webperside/courseerpbackend/repository/TeacherRepository.java:15]: but it is recommended :)

**Review by hamidsultanzadeh — APPROVED** [review]

**Pull request merged**